### PR TITLE
Create stepper skeleton

### PR DESCRIPTION
**Pull request by LizBaker**
This is a WIP for an upcoming component. For the first round of review we can ignore css, mostly we want to review the structure of the step mdx and its translation process

### CODE REVIEW

File: src/homepageSteps/step1.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+homePage: true
```
Comment (tabathadelane) [inline src/homepageSteps/step1.mdx:2]: What is this formatter for? I can't find it referenced anywhere else.

**Review by tabathadelane — DISMISSED** [review]
My only question would be about the `homePage` frontmatter being used to describe this. Is this just an early way to differentiate between 'Stepper' folders if in the future there are multiple?

**Comment by LizBaker** [comment]
> My only question would be about the `homePage` frontmatter being used to describe this. Is this just an early way to differentiate between 'Stepper' folders if in the future there are multiple?

@tabathadelane that gets referenced [here](https://github.com/newrelic/docs-website/pull/11229/files#diff-26ca35c3be6d07c84bb77f1e63a1b19487279d5b9c9a9392179b3a35647e1254R224) so we can query/filter for just the step mdx

File: scripts/actions/fetch-and-deserialize.js [hunk 2]
```diff
@@ -130,23 +129,25 @@ const fetchTranslatedFilesZip = (locale) => {
...
+      .map((entry) => {
+        if (entry.entryName.endsWith('.mdx')) {
+          return {
+            path: entry.entryName,
+            html: zip.readAsText(entry, 'utf8'),
+          };
+        }
+        return null;
+      })
+      .filter(Boolean);
```
Comment (sunnyzanchi) [inline scripts/actions/fetch-and-deserialize.js:150]:
```suggestion
      .filter((entry) => entry.entryName.endsWith('.mdx'))
      .map((entry) => ({
        path: entry.entryName,
        html: zip.readAsText(entry, 'utf8'),
      }))
```

**Review by sunnyzanchi — DISMISSED** [review]
left one suggestion but otherwise looks good to me

File: scripts/actions/fetch-and-deserialize.js (hunk 2 above)
Comment (LizBaker) [reply scripts/actions/fetch-and-deserialize.js:150]: ooooh nice

**Comment by LizBaker** [comment]
cache clear build: [build-12d43ac8-c4c2-4de4-ac93-8cb0d7c28f3a.gatsbyjs.io](https://build-12d43ac8-c4c2-4de4-ac93-8cb0d7c28f3a.gatsbyjs.io/)

**Comment by tabathadelane** [comment]
> > My only question would be about the `homePage` frontmatter being used to describe this. Is this just an early way to differentiate between 'Stepper' folders if in the future there are multiple?
> 
> @tabathadelane that gets referenced [here](https://github.com/newrelic/docs-website/pull/11229/files#diff-26ca35c3be6d07c84bb77f1e63a1b19487279d5b9c9a9392179b3a35647e1254R224) so we can query/filter for just the step mdx

I was just curious how that might carry over into future uses like maybe within a tutorial-like doc, but I think I am seeing that is a way to query the individual uses, so this would be fine.

**Review by tabathadelane — APPROVED** [review]

**Pull request closed without merging**